plugins/right-sizer/src/controller/controller.go

```diff
@@ -67,6 +68,14 @@ func WithAllowedNamespaces(value []string) controllerOption {
 	}
 }
 
+// WithAllowedUpdateNamespaces sets the corresponding field in a controllerConfig type.
+func WithAllowedUpdateNamespaces(value []string) controllerOption {
+	return func(c *controllerConfig) {
+		c.allowedUpdateNamespaces = value
```

Comment:
do we use `allowedNamespaces` as the default if `allowedUpdateNamespaces` is unspecified? I'm not seeing that logic.

I'd imagine something like
```go
func WithAllowedNamespaces(value []string) controllerOption {
  return func(c *controllerConfig) {
    c.allowedNamespaces = value
    if c.allowedUpdateNamespaces == nil {
      c.allowdedUpdateNamespaces = value
    }
  }
}
```

Reply:
The global namespaces aren't used as the default, only because the check of global namespaces happens earlier in logic and short-circuits the check of update-namespaces: IF the global namespaces is set, and a given namespace isn't in that list, it skips processing that pod, including caring about whether limits should be updated, Etc.